tensorboard/webapp/metrics/views/right_pane/settings_view_component.ng.html

```diff
@@ -37,20 +37,21 @@ <h3 class="section-title">Scalars</h3>
       <mat-slider
         aria-labelledby="scalars-smoothing-label"
         color="primary"
-        [max]="1"
+        [max]="MAX_SMOOTHING_VALUE"
         [min]="0"
-        [step]="0.01"
+        [step]="0.001"
```

Comment:
Do we absolutely need the triple precision? It looks really bad on the fob. I think it can stay `0.01` and max `0.99` (and revert the formatting which I think is a bit janky) unless people are giving us feedback otherwise.

Reply:
I think you're right, the extra precision is likely too aggressive, and yeah, the formatting fn is unfortunate.

Reduced to max=0.99 with step=0.01